### PR TITLE
Feature 1 login

### PR DESCRIPTION
**Pull request by almedaria**
Please review feature 1 log in

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "axios": "^0.21.4",
```
Comment (ccases) [inline package.json:9]: Good add! axios is very useful

File: src/components/LogIn/LogIn.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import React, { useContext, useState, useEffect } from "react";
+import axios from "axios";
+import APIHeaders from "../../APIContext";
+
+function LogIn() {
+  const [username, setUsername] = useState("");
+  const [pw, setPw] = useState("");
+  const [header, setHeader] = useContext(APIHeaders);
+  useEffect(() => {
```
Comment (ccases) [inline src/components/LogIn/LogIn.js:9]: Great use of `useEffect`!

File: package.json (hunk 1 above)
Comment (kbcacas) [reply package.json:9]: Looks good to me

File: src/components/LogIn/LogIn.js (hunk 2 above)
Comment (kbcacas) [reply src/components/LogIn/LogIn.js:9]: Great!

**Pull request merged**